README.md

```diff
@@ -6,12 +6,62 @@ Web service wrapper for [fqm-execution](https://github.com/projecttacoma/fqm-exe
+  as well as `ValueSet` objects with code expansions included. 
+* `patients`: an array of FHIR `Bundle` resources, each of which contains the `Patient` object and any required clinical resources (`Condition`, `Medication`, etc.)
+  for measure calculation. This must be an array, even if only one patient is provided.
+  * NOTE: `/Measure/$care-gaps` only accepts 1 patient in this array currently.
+* `options`: an optional object containing Calculation Options, each of which is listed below, with its default.
+
+#### Calculation Options
+
+The options that we support for calculation are as follows:
+| option                 |  type   | optional? | description                                                                           |
+| :--------------------- | :-----: | :-------: |           :-------------------------------------------------------------------------- |
+| includeClauseResults   | boolean |    yes    |                                  Option to include clause results. Defaults to false. |
+| includePrettyResults   | boolean |    yes    |             Option to include pretty results on statement results. Defaults to false. |
+| includeHighlighting    | boolean |    yes    |                   Include highlighting in MeasureReport narrative. Defaults to false. |
+| measurementPeriodStart | string  |    yes    | Start of measurement period. Defaults to the period provided in the `Measure` resource|
+| measurementPeriodEnd   | string  |    yes    |  End of measurement period. Defaults to the period provided in the `Measure` resource |
+| calculateSDEs          | boolean |    yes    |                Include Supplemental Data Elementss in calculation. Defaults to false. |
+| calculateHTML          | boolean |    yes    |                           Include HTML structure for highlighting. Defaults to false. |
+
+These types will be passed through to the calculation service, and will be used in the execution of the measure.
+
+### Request Outputs
+
+Each request output has a different output format, based on the data being conveyed:
+* `/calculate`: an array of `ExecutionResult` objects. Each `ExecutionResult` object contains:
+    * `patient`: the ID of the patient this object is associated with
+    * `detailedResults`, a map of `group`s to an object which contains population-, statement-, and clause-level results for that group, as well as the html
+      for that group (if requested by the provided options)
+    * `evaluatedResource`: an array of the FHIR resources used in the calculation of the measure for this patient
+    * `supplementalData`: if requested by the provided options, an array containing the raw and HTML structure results for each supplemental data element in the measure, 
+      calculated for the specified patient
+* `/calculateRaw`: a `cql.Result` object, which contains: 
+   * `patientResults`: A map of patient IDs to Statement-level results for that patient
```

Comment:
Can you add more detail on "statement-level results", or at least an example

Reply:
Added